source/server-discovery-and-monitoring/tests/integration/minPoolSize-error.json

```diff
@@ -0,0 +1,104 @@
+{
+  "runOn": [
+    {
+      "minServerVersion": "4.4"
+    }
+  ],
+  "database_name": "sdam-tests",
+  "collection_name": "sdam-minPoolSize-error",
+  "data": [],
+  "tests": [
+    {
+      "description": "Command error on minPoolSize background creation",
+      "failPoint": {
+        "configureFailPoint": "failCommand",
+        "mode": {
+          "skip": 2
+        },
+        "data": {
+          "failCommands": [
+            "isMaster"
+          ],
+          "appName": "SDAMminPoolSizeError",
+          "closeConnection": false,
+          "errorCode": 91
+        }
+      },
+      "clientOptions": {
+        "heartbeatFrequencyMS": 10000,
+        "appname": "SDAMminPoolSizeError",
+        "minPoolSize": 10,
+        "serverSelectionTimeoutMS": 100
+      },
+      "operations": [
+        {
+          "name": "waitForEvent",
+          "object": "testRunner",
+          "arguments": {
+            "event": "PoolReadyEvent",
+            "count": 1
+          }
+        },
+        {
+          "name": "waitForEvent",
+          "object": "testRunner",
+          "arguments": {
+            "event": "PoolClearedEvent",
+            "count": 1
+          }
+        },
+        {
+          "name": "waitForEvent",
+          "object": "testRunner",
+          "arguments": {
+            "event": "ServerMarkedUnknownEvent",
+            "count": 1
+          }
+        },
+        {
+          "name": "runCommand",
+          "object": "database",
+          "command_name": "ping",
+          "arguments": {
+            "command": {
+              "ping": {}
+            }
+          },
+          "error": false
```

Comment:
This needs to be `"error": true`.  The yaml file is correct and this json file is out of sync.

Reply:
fixed